### PR TITLE
fix(breakdowns): insight breakdowns paging order is non-stable

### PR DESCRIPTION
**Pull request by pauldambra**
## Changes

In [this slack thread](https://posthog.slack.com/archives/C02LR7352SG/p1643738897887099) a customer reports that they see duplicated records when they click "load more breakdowns" on an insight

The SQL that selects breakdown values was not stable since it only order by count. This meant we could return the same data in two pages of results.

## How did you test this code?

Running tests and clicking around in the UI

### CODE REVIEW

**Comment by neilkakkar** [comment]
ofcourse the fix broke all snapshots, sigh. (Can I leave this with you for whenever you get to it? 🙈 )

**Comment by neilkakkar** [comment]
Fun update: I could reproduce Joe's bug on my personal PostHog Cloud project. No clue why it doesn't happen on our team, but I suspect once this goes in, we'll start seeing Joe's issue on our Cloud account as well. At which point the fix will be to change the offset from 20 to 25 :P

**Comment by guidoiaquinti** [comment]
FYI here's an additional [customer report](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1644916933462829).

**Comment by neilkakkar** [comment]
Remaining tests seem to have different behaviour on different CH versions (pass locally, but not on CI?). Will pause this until after CH update.

**Comment by pauldambra** [comment]
we want to keep this open until after CH upgrade

**Comment by pauldambra** [comment]
Still want to wait until CH upgrade before resolving this

**Comment by neilkakkar** [comment]
New CH version is here, so I might look into completing this whenever I get a second

File: ee/clickhouse/views/test/__snapshots__/test_clickhouse_experiments.ambr [hunk 1]
```diff
@@ -12,8 +12,8 @@
        AND timestamp <= toDateTime('2020-01-06 00:00:00')
        AND (has(['control', 'test'], replaceRegexpAll(JSONExtractRaw(e.properties, '$feature/a-b-test'), '^"|"$', '')))
      GROUP BY value
-     ORDER BY count DESC
-     LIMIT 25
+     ORDER BY count DESC, value DESC
+     LIMIT 10
```
Comment (neilkakkar) [inline ee/clickhouse/views/test/__snapshots__/test_clickhouse_experiments.ambr:16]: there's no reason for this to happen 🤔

File: ee/clickhouse/queries/test/test_trends.py [hunk 2]
```diff
@@ -99,9 +99,9 @@ def test_breakdown_with_filter_groups(self):
         )
 
         self.assertEqual(len(response), 2)
-        self.assertEqual(response[0]["breakdown_value"], "oh")
+        self.assertEqual(response[0]["breakdown_value"], "uh")
```
Comment (neilkakkar) [inline ee/clickhouse/queries/test/test_trends.py:102]: also opposite of what it should be, need to investigate

**Comment by neilkakkar** [comment]
superceded by https://github.com/PostHog/posthog/pull/10050

**Pull request closed without merging**